src/tasks/dimensionality_reduction/metrics/rmse/script.py

```diff
@@ -31,12 +31,13 @@ def _rmse(X, X_emb):
 X_emb = input_embedding.obsm["X_emb"]
 
 print("Compute NNLS residual after SVD", flush=True)
-n_svd = 200
+n_svd = 500
 svd_emb = sklearn.decomposition.TruncatedSVD(n_svd).fit_transform(high_dim)
 rmse = _rmse(svd_emb, X_emb)
 
+#! TODO: change spectral to commit 80b37e7a6aa27df4436f400397564c01276817e0
 print("Compute NLSS residual after spectral embedding", flush=True)
```

Comment:
in openproblems v1 commit 80b37e7a6aa27df4436f400397564c01276817e0 this has changed to use the diffusion map method. @rcannood can you have a look on how to proceed?